qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -93,6 +107,10 @@ Optional fields for centralized Qiita by portal
 Prep information file
 ---------------------
 
+The *prep information file* will describe the sequencing technology or analytical
```

Comment:
Note that sequencing is correct but we are also working on other preparation files, like metabolomics.